config/samples/enclave-cc/sim/kustomization.yaml

```diff
@@ -5,6 +5,6 @@ nameSuffix: -sgx-mode-sim
 
 images:
 - name: quay.io/confidential-containers/runtime-payload-ci
-  newTag: enclave-cc-SIM-sample-kbc-b58c59dc72a0b10ae3dd4436a206fd54a820c9f7
+  newTag: enclave-cc-SIM-sample-kbc-753599f76ef201968438b15f01d7b37d49cc7c5c
```

Comment:
... and this one, could you use ed979ef2952d8b20c2f0abee3d32a3617d2c57bb tag?

enclave-cc merged a bump on Occlum Today, and we want to test with that image.